### ISSUE TITLE
MP4 media is not playing when the server responds with chuncked data and http header attribute Content-Encoding: identity

### ISSUE THREAD

**Issue by a-schade** [Igalia/cog#463]
When the server sends the mp4 as chuncked data and the http header attribute content-encoding is set to identity the mp4 file is not played. 

- Yocto linux arm64
- Wayland/Weston
- i.mx8x
- Cog Version: 0.12.4 (WPE WebKit 2.36.3)
- Server: embedio 3.4.3

In the console is just see that AIUR is started

`====== AIUR: 4.5.7 build on Nov 13 2020 08:36:18. ======
        Core: MPEG4PARSER_06.17.17  build on Nov 12 2020 05:13:57
 file: /usr/lib/imx-mm/parser/lib_mp4_parser_arm_elinux.so.3.2`

and then nothing.

I checked with wireshark that the data was realy send by the server.

When i remove the attribute and send the data as a whole the video is played. When i tried to check this behaviour against chrome i saw that chrome requests the media as partial content (206).

A similar problem arise when webm media should be played. This time the media is requested as "206", but also get stuck at AIUR.

**Comment by philn** [comment]
Is this playing fine with `gst-play-1.0` ?
Also please note: we do not provide official support about NXP things.

**Comment by a-schade** [comment]
Yes, the media file is playing fine with gst-play-1.0.

**Comment by philn** [comment]
Quoting RFC 2616, https://www.rfc-editor.org/rfc/rfc2616#section-3.5 :

> identity
>        The default (identity) encoding; the use of no transformation
>        whatsoever. This content-coding is used only in the Accept-
>        Encoding header, and SHOULD NOT be used in the Content-Encoding
>        header.

So, not sure this is a bug. This header is invalid, AFAICS.

**Comment by a-schade** [comment]
You are right i mixed it up with the accept-encoding header. So this point goes the server devs. I will double check that it still won't work when identity is removed and only transfer-encoding is chuncked set.

**Comment by a-schade** [comment]
When i remove the wrong Content-Encoding header and send the data as chuncked. The problem persists.

Response

> Hypertext Transfer Protocol
>     HTTP/1.1 200 OK\r\n
>     Vary: Accept-Encoding\r\n
>     ETag: "5kIBpjFF2Qg0V8MAAAAAAA"\r\n
>     Last-Modified: Mon, 12 Jul 2021 12:36:19 GMT\r\n
>     Cache-Control: max-age=0, must-revalidate\r\n
>     Accept-Ranges: bytes\r\n
>     Content-Type: video/mp4; charset=utf-8\r\n    
>     Date: Wed, 13 Jul 2022 08:24:51 GMT\r\n
>     Connection: close\r\n
>     Transfer-Encoding: chunked\r\n
>     \r\n
>     [HTTP response 4/4]
>     [Time since request: 0.053133000 seconds]
>     [Prev request in frame: 2801]
>     [Prev response in frame: 2802]
>     [Request in frame: 2836]
>     [Request URI: http://192.168.178.49/videos/video.mp4]
>     HTTP chunked response
>     File Data: 810961 bytes
